Makefile

```diff
@@ -21,8 +21,8 @@ ARCH_ALIAS_x86_64 = amd64
 ARCH_ALIAS_aarch64 = arm64
 ARCH_ALIAS = $(shell echo "$(ARCH_ALIAS_$(ARCH))")
 
-NERDCTL_VERSION=1.0.0
-QEMU_VERSION=v6.2.0
+NERDCTL_VERSION=1.1.0
+QEMU_VERSION=v7.0.0
```

Comment:
Can we just use v7.1.0 ?
https://hub.docker.com/layers/tonistiigi/binfmt/buildkit-v7.1.0/images/sha256-3dd2341845b2761c4ec41225e609958380c55e7933e922d1f769b6871e52621c?context=explore

Reply:
I don't know; what is the difference between the `qemu-*` and the `buildkit-*` tags?

So far we always copied from the `qemu-*` tags, and there is no `qemu-7.1.0` image.